Dockerized makefile default target doesn't exist

```
make -f docker.Makefile
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
make: *** No rule to make target `bin', needed by `all'.  Stop.
```

Specifically the default target is set to `bin test`, but there is no top level `bin` target. Looks like `cross` might be the relevant target?

--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,7 @@ include vars.mk
 GO_BUILD := CGO_ENABLED=0 go build
 GO_TEST := CGO_ENABLED=0 go test
 
-all: bin test
+all: bin/$(BIN_NAME) test
 
 check_go_env:
 	@test $$(go list) = "$(PKG_NAME)" || \

--- a/docker.Makefile
+++ b/docker.Makefile
@@ -10,7 +10,7 @@ IMAGE_BUILD_ARGS := \
 PKG_PATH := /go/src/$(PKG_NAME)
 
 .DEFAULT: all
-all: bin test
+all: bin/$(BIN_NAME) test
 
 create_bin:
 	@mkdir -p bin

